docker-tomcat.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+exec docker run -it --rm \
+            --link $(docker-compose ps -q postgresql):postgresql \
+            -v $PWD/target/sigmah-2.2-SNAPSHOT/:/usr/local/tomcat/webapps/sigmah/:ro \
```

Comment:
Please, remove hard-coded `2.2-SNAPSHOT`.

You can start your script with something like

```bash
# gives SIGMAH_VERSION="2.2-SNAPSHOT"
SIGMAH_VERSION=`mvn --quiet -Dexec.executable='echo' -Dexec.args='${project.version}' --non-recursive exec:exec`
```

See also https://stackoverflow.com/questions/3545292/how-to-get-maven-project-version-to-the-bash-command-line